jetstream/cli.py

```diff
@@ -340,23 +421,30 @@ def convert(self, value, param, ctx):
 @experiment_slug_option
 @bucket_option
 @secret_config_file_option
+@recreate_enrollments_option
 def run(
     project_id,
     dataset_id,
     date,
     experiment_slug,
     bucket,
     config_file,
+    recreate_enrollments,
 ):
     """Runs analysis for the provided date."""
-    success = AnalysisExecutor(
+    analysis_executor = AnalysisExecutor(
         project_id=project_id,
         dataset_id=dataset_id,
         bucket=bucket,
         date=date,
         experiment_slugs=[experiment_slug] if experiment_slug else All,
         configuration_map={experiment_slug: config_file} if experiment_slug and config_file else {},
-    ).execute(strategy=SerialExecutorStrategy(project_id, dataset_id, bucket))
+    )
+
+    analysis_executor.ensure_enrollments(recreate_enrollments=recreate_enrollments)
+    success = analysis_executor.execute(
+        strategy=SerialExecutorStrategy(project_id, dataset_id, bucket, recreate_enrollments)
+    )
```

Comment:
Why is ensuring enrollments a separate step from executing? Isn't it always a prerequisite?